src/middlewared/middlewared/plugins/truenas_connect/acme.py

```diff
@@ -38,6 +38,16 @@ async def update_ui(self):
             # Let's restart UI now
             # TODO: Hash this out with everyone
             await self.middleware.call('system.general.ui_restart', 2)
+            if await self.middleware.call('system.is_ha_capable'):
```

Comment:
All of our HA platforms that are sold as single-node systems will always raise an exception here. We need to check if the system has an HA license.
```suggestion
            if await self.middleware.call('failover.licensed'):
```